tests/integration/test_github.py

```diff
@@ -239,6 +238,20 @@ def test_pr_info(self):
         assert pr_info.title.startswith("new")
         assert pr_info.status == PRStatus.merged
 
+    def test_all_pr_commits(self):
+        commits = self.colin_project.get_all_pr_commits(pr_id=1)
+        assert commits
+        assert len(commits) == 7
+        assert commits == [
```

Comment:
I think you don't need the first two asserts since the third implicates them.

Reply:
You are right with `len(commits)` but in whole `tests` you can see similar tests.
https://github.com/packit-service/ogr/blob/master/tests/integration/test_github.py#L62 https://github.com/packit-service/ogr/blob/master/tests/integration/test_github.py#L240

Then I would prefer to rewrite `test_github.py` so we have the same or similar tests code structure.